### DISCUSSION TITLE
RaspberryMatic ist plötzlich nicht mehr erreichbar, Smarthome funktioniert aber noch

### DISCUSSION THREAD

**Discussion by codingcoast** [OpenCCU/OpenCCU#1527] [🇩🇪 Fragen&Antworten]
Hallo zusammen,
ich habe ein sehr mysteriöses Phänomen:
RaspberryMatic lief auf meiner CCU3 jetzt etwa 6 Wochen.
Seit gestern blinkt die CCU3 (Das bedeutet sie hat kein Internet mehr) und Ich komme auch nicht ins Webinterface (Connection timeout). In meinem Router ist die CCU verbunden, aber in den Logs zeigt er alle 10 Sekunden **DHCPDISCOVER** und **DHCPOFFER** (So weit ich weiß bietet der Router der CCU3 eine IP an, aber sie wird nicht angenommen?)
Die IP lässt sich auch nicht anpingen.
Aber: Mein Smarthome funktioniert noch problemlos, ich kann über Apple Homekit die Aktoren weiterhin steuern und sehe den Status.

Dann habe ich die CCU3 mit einem Netzwerkkabel direkt mit dem Laptop verbunden.
Über ifconfig die IP rausgefunden, aber auch hier bleibt ein Ping erfolglos, von SSH ganz zu schweigen.
Hat jemand eine Idee wie ich auf die CCU komme? Ich vermute da ist irgendwas in den Netzwerkeinstellungen faul.
Evtl. ist noch erwähnenswert, dass ich meinen Router vor einigen Tagen von einem Telekom Speedport Smart 3 auf einen Asus RT-AX86u gewechselt habe. Die CCU war unter dem neuen Router aber schon erreichbar.

Mit USB komme ich auch nicht drauf, zumindest wüsste ich nicht was ich da probieren kann.
Ich verstehe nicht, warum die Verbindung zum AppleTV (Homekit) problemlos fuktioniert, aber ich nichtmal die IP anpingen kann, weder über den Router, noch direkt am PC.

Ich bin am Ende, ist die Ethernet Schnittstelle meiner CCU defekt?

```
:ifconfig | grep broadcast
inet 169.254.138.48 netmask 0xffff0000 broadcast 169.254.255.255

:ping 169.254.255.255
PING 169.254.255.255 (169.254.255.255): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
ping: sendto: No route to host
Request timeout for icmp_seq 2
ping: sendto: Host is down
Request timeout for icmp_seq 3
ping: sendto: Host is down
Request timeout for icmp_seq 4
ping: sendto: Host is down
Request timeout for icmp_seq 5
ping: sendto: Host is down
Request timeout for icmp_seq 6
```

**Comment by SaschaPeter** [comment]
Hi codingcoast, also zumindest zu der Direktverbindung kann ich dir sagen, das funktioniert so nicht.
Die 169.254.x.x (APIPA) erhält dein Laptop, weil er den DHCP nicht erreichen kann, wie hast du versucht die IP deiner ccu herauszufinden, wenn sie vom DHCP(Router) keine IP erhält? Und als letztes, um eine direkte Verbindung per Ethernetkabel herzustellen, bräuchtest du ein Crossover-Kabel und eine statische IP-Konfiguration auf beiden Geräten.
Du hast bisher auch nicht viel geschrieben, ob du gestern irgendetwas geändert hast z.B. ein Update eingespielt.

Wie sieht es mit einem Backup aus?
Ist ssh aktiviert, hast du mal probiert einen Monitor und eine Tastatur anzuschließen?

**Comment by codingcoast** [comment]
Danke für die Antwort.
Stimmt, die Direktverbindung kann ich so vergessen.

Ne, ich habe kein Update eingespielt und die CCU3 war völlig unangetastet. Das einzige ist der Router Wechsel. Aber es hatte ja mit dem neuen funktioniert und es macht auch sonst kein Gerät dieses Problem.
Ein Backup / SSH ist ja nicht möglich da ich nicht drauf komme. Der Router bietet zwar eine IP an, aber die wird laut Logs nicht angenommen.
Die IP lässt sich auch nicht anpingen, kein SSH, kein Webinterface.

Monitor und Tastatur? Ich habe da nur ein Ethernet und 2 USB Ports.

Hier mal die Logs aus dem Router:

```
Nov 15 20:30:11 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:11 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:15 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:15 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:18 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:18 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:21 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:21 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:24 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:24 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:27 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:27 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:30 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:30 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:33 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:33 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:36 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:36 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:39 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:39 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:42 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:42 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
Nov 15 20:30:45 dnsmasq-dhcp[1573]: DHCPDISCOVER(br0) b8:27:eb:ac:7a:dc 
Nov 15 20:30:45 dnsmasq-dhcp[1573]: DHCPOFFER(br0) 192.168.50.105 b8:27:eb:ac:7a:dc 
```
